setup.cfg

```diff
@@ -32,13 +32,13 @@ install_requires =
     nibabel >=3.0.1
     nipype >=1.5.1,<2.0
     traits <6.4
-    niworkflows ~= 1.6.0
+    niworkflows @ git+https://github.com/nipreps/niworkflows.git@master
```

Comment:
I just cut a new version, this can be set to `~= 1.6.3`

Reply:
Were we not going to use bleeding edge pinning for dev branches, and limited pinning when about to cut a new release?

Reply:
```suggestion
    niworkflows ~= 1.6.3
```